cms/djangoapps/contentstore/helpers.py

```diff
@@ -375,10 +385,20 @@ def _import_xml_node_to_parent(
     if copied_from_block:
         # Store a reference to where this block was copied from, in the 'copied_from_block' field (AuthoringMixin)
         temp_xblock.copied_from_block = copied_from_block
+    if copied_from_block and link_to_upstream:
+        # If requested, link this block as a downstream of where it was copied from
+        try:
+            validate_upstream_key(copied_from_block)
+        except UnsupportedUpstreamKeyType:
+            pass  # @@TODO - should we let this error bubble up?
```

Comment:
This would get called during course import, right? If so, maybe just log this as an error for now? At some point we may add other supported key types that people will have in their Teak-generated content exports. Sumac might not recognize those or be able to hook it up correctly when importing, but it should try not to explode the import process.

Reply:
This is no longer necessary. The OLX has all the information the linked content needs.